lib/Converter/FunctionConverter.cpp

```diff
@@ -90,6 +90,7 @@ void FunctionConverter::convertOutputs(Node &node) {
     //   "conversion".
     // 3. Node may be a module level value and we need one conversion per
     //    function.
+    size_t index = 0;
```

Comment:
I'm a bit confused by what `index` is here. First off you're only incrementing it when you get past a couple continues in the body of the loop. Second even if it was incremented at the top of the loop, it represents which user number this is, is that right? I'm not sure why this is relevant?

Reply:
thx for bringing the comment, changed to use ResNo instead